docs/typedoc.mdx

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
This file will be expanded on, but doesn't appear in the sidebar of our docs yet